CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 - Enhanced accuracy of time-to-display spans. ([#4189](https://github.com/getsentry/sentry-react-native/pull/4189))
 
+### Changes
```

Comment:
l: I think Features is a better fit. We use Changes mainly in major version for breaking changes and others.